webapp-django/crashstats/api/jinja2/api/documentation.html

```diff
@@ -1,152 +1,139 @@
-{% extends "crashstats_base.html" %}
-
-{% block page_title %}API Documentation{% endblock %}
-
-{% block product_nav_filter %}&nbsp;{% endblock %}
```

Comment:
The API docs used to live in their own little world. This PR unifies them with all the other docs so they have the same nav header and such.